### ISSUE TITLE
Discuss if we want audit-tool beta on Linux servers

### ISSUE THREAD

**Issue by jimk-bdrc** [buda-base/asset-manager#166]
And install

**Comment by jimk-bdrc** [comment]
Note I've updated bodhi:~service/.config/bdrc/auditTool/user.properties` to include the new EXIF Thumbnail error for Archive (thumbnails in images will fail)

# bodhi 
has the latest beta as the default:

```
$ update-alternatives --config audit-tool
There are 3 choices for the alternative audit-tool (providing /usr/local/bin/audit-tool).

  Selection    Path                                          Priority   Status
------------------------------------------------------------
  0            /opt/audit-v1/bin/audit-tool                   90        auto mode
* 1            /opt/audit-tool/audit-v1-beta/bin/audit-tool   80        manual mode
  2            /opt/audit-tool/audit-v101/bin/audit-tool      50        manual mode
  3            /opt/audit-v1/bin/audit-tool                   90        manual mode

Press <enter> to keep the current choice[*], or type selection number:
```

and this is the most recent checkin:

```
$ audit-tool -version                                                             
Version 1.0Beta_2022_04_18
```

# sattva
 the DIP-pump and IA-upload host, has the earlier v1.0-beta from 28 Jan, 2022

```
$ update-alternatives --config audit-tool                                                                                       ─╯
There are 2 choices for the alternative audit-tool (providing /usr/local/bin/audit-tool).

  Selection    Path                                          Priority   Status
------------------------------------------------------------
  0            /opt/audit-v1/bin/audit-tool                   500       auto mode
  1            /opt/audit-tool/audit-v1-beta/bin/audit-tool   400       manual mode
* 2            /opt/audit-v1/bin/audit-tool                   500       manual mode

Press <enter> to keep the current choice[*], or type selection number:

$  ls -l  /opt/audit-v1/bin/audit-tool                                                          
-rwxr-xr-x 1 root root 17320 Jan 28 14:43 /opt/audit-v1/bin/audit-tool
```

**Comment by jimk-bdrc** [comment]
@TBRC-Travis @TBRC-TimB I updated `bodhi` to version 1.0Beta-2022-04-25 which has
- fix for #127 image file name correctness
- fix for #162 rename log to WARN-Work when tests PASS or WARN
-

**Comment by jimk-bdrc** [comment]
We do, and we have
